lib/discourse_code_review/importer.rb

```diff
@@ -33,17 +33,22 @@ def import_commits
       end
     end
 
-    def auto_link_commits(text)
+    def auto_link_commits(text, doc = nil)
       linked_commits = find_linked_commits(text)
       if (linked_commits.length > 0)
+        cooked = PrettyText.cook(text)
```

Comment:
If you already have a `doc`, then there's no point in cooking the post.

I would do something like 

``` ruby
doc ||= Nokogiri::HTML::fragment(PrettyText.cook(text))
```